charts/kubestash-operator/templates/operator/deployment.yaml

```diff
@@ -47,6 +47,9 @@ spec:
         {{- if include "appscode.licenseSecretName" . }}
           - --license-file=/var/run/secrets/appscode/license/key.txt
         {{- end }}
+        {{- if and .Values.global .Values.global.networkPolicy.enabled }}
+          - --enable-network-policy=true
+        {{- end }}
```

Comment:
Set this in `configuration.yaml`.
Following this change: https://github.com/kubestash/apimachinery/pull/136